cfme/tests/intelligence/test_chargeback.py

```diff
@@ -19,11 +19,17 @@
     test_requirements.chargeback
 ]
 
-per_time = ['Hourly', 'Daily', 'Monthly', 'Weekly', 'Yearly'] if current_version() >= '5.7' \
-    else ['Hourly', 'Monthly']
 
+@pytest.fixture
```

Comment:
**REQUIRED:** I don't see a need for this to be a fixture, it can just be a standard function